src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/appointments/AppointmentsMigrationService.kt

```diff
@@ -163,3 +160,26 @@ class AppointmentsMigrationService(
     updatedBy = modifiedBy,
   )
 }
+
+private val simpleTimeFormat = DateTimeFormatter.ofPattern("HH:mm")
+
+data class AppointmentResponse(
+  val bookingId: Long,
+  val offenderNo: String,
+  // prison or toPrison is never null in existing nomis data for event_type = 'APP' (as at 11/5/2023)
+  val prisonId: String,
+  val internalLocation: Long? = null,
+  val startDateTime: LocalDateTime? = null,
+  val endDateTime: LocalDateTime? = null,
+  val comment: String? = null,
+  val subtype: String,
+  val status: String,
+  val createdDate: LocalDateTime,
+  val createdBy: String,
+  val modifiedDate: LocalDateTime? = null,
+  val modifiedBy: String? = null,
+)
```

Comment:
I previously replaced this with the swagger generated model without comparing the two - that was a mistake 😢